addons/OQ_Toolkit/plugin.gd

```diff
@@ -4,7 +4,13 @@ extends EditorPlugin
 
 func _enter_tree():
 	add_autoload_singleton("vr", "res://addons/OQ_Toolkit/vr_autoload.gd")
+	
+	add_custom_type("OQ_ARVROrigin", "ARVROrigin", load("res://addons/OQ_Toolkit/OQ_ARVROrigin/scripts/OQ_ARVROrigin.gd"), load("res://addons/OQ_Toolkit/oqt_icon.png"))
```

Comment:
I'm not sure if making some classes as build in types improves the usability of the toolkit; my main concerns are:

- If not all classes are a custom type it is inconsistent and not clear to a user when to use what
- I would assume that in most cases a user will not need to add nodes of the OQ_Toolkit very often but just setup the basic VR requirements and is done (especially for the origin, camera and controller)
- The folder structure is actually part of the documentation as it makes clear which features should be attached to which node type; making everything a custom class would loose this (and I don't like to add a lot of tool-code to the scripts to check for this at editing time as this will get unreadable and hard to maintain quickly)